src/nvim/os/env.c

```diff
@@ -346,6 +349,17 @@ void init_homedir(void)
   }
```

Comment:
maybe `uv_os_homedir` also works on Windows, then we could try it before fallback to `C:/` ?

Reply:
Done